pkg/kubelet/rkt/rkt.go

```diff
@@ -349,9 +350,26 @@ func setApp(app *appctypes.App, c *api.Container) error {
 	return setIsolators(app, c)
 }
 
+// getImageManifest invokes 'rkt image cat-manifest' to retrive the image manifest
+// for the image.
+func (r *runtime) getImageManifest(image string) (*appcschema.ImageManifest, error) {
+	var manifest appcschema.ImageManifest
+
+	// TODO(yifan): Assume docker images for now.
+	output, err := r.runCommand("image", "cat-manifest", "--quiet", dockerPrefix+image)
+	if err != nil {
+		return nil, err
+	}
+	if len(output) != 1 {
+		return nil, fmt.Errorf("no output")
```

Comment:
It shouldn't, I redirect the stdout to null in rkt in that PR. But I guess "invalid output + the real output" sounds better. Thank you!